### ISSUE TITLE
dates syp is depositing are still wrong

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#1623]
It looks like SYP is still getting dates wrong. Natalia has been cataloguing this: https://docs.google.com/spreadsheets/d/1NR93P-qStFxZVsrHyqqBJ1TscWsEEgJh5sS7ib-RZSE/edit#gid=0 and how to fix it. Would be great to fix this.

**Comment by markmacgillivray** [comment]
Checking the ones in that sheet, the new system gives more accurate matches. However it is still different in some cases, and those appear to be because in the sheet it looks like you're assuming any earlier date is the best one, whereas previously you've said that the "print" date overrules the "online" date, and I believe we did that because there can be many where the online date is far later than the print date (although some of the examples in the sheet are where it is earlier).

So I can make changes for this if you want, but you'll need to decide what you want those changes to be. In some cases you also appear to be prioritising the journal published date over the main published date. So just let me know what order you'd like those to take precedence in now, and I can change for that.

**Comment by JosephMcArthur** [comment]
Bumping this up the priority list as in the finance work we saw several issues with dates.

Here are some examples:

10.1093/cid/ciaa002
10.12688/gatesopenres.13071.1
10.1111/1365-2435.13683
10.1021/acs.biomac.0c00539
10.1038/s41437-020-00353-1
10.1093/cid/ciaa076
10.1038/s41370-019-0177-7
10.1007/s11162-019-09582-8
10.1177/0895904818813302
10.1080/13229400.2018.1511451

All those had a `year` of 2021 but their `date` was pre-2021.

**Comment by JosephMcArthur** [comment]
Bumping this up the priority list as in the finance work we saw several issues with dates.

Here are some examples:

10.1093/cid/ciaa002
10.12688/gatesopenres.13071.1
10.1111/1365-2435.13683
10.1021/acs.biomac.0c00539
10.1038/s41437-020-00353-1
10.1093/cid/ciaa076
10.1038/s41370-019-0177-7
10.1007/s11162-019-09582-8
10.1177/0895904818813302
10.1080/13229400.2018.1511451

All those had a `year` of 2021 but their `date` was pre-2021.

**Comment by JosephMcArthur** [comment]
@markmacgillivray any chance you could take a look at this again early this week please once the EUPMC stuff is in good shape?

**Comment by markmacgillivray** [comment]
I believe this is fixed on new dev.

**Comment by markmacgillivray** [comment]
Nothing new to report here, as mentioned above, I think this is fixed now by other work.

**Comment by JosephMcArthur** [comment]
For non-oareport systems, are those dates now coming from unpaywall too?

**Comment by markmacgillivray** [comment]
Not necessarily, it depends on what data we find first, and other things can take precedence over unpaywall. However all systems are taking account of the changes I made to how we calculate them from crossref. I believe you said on Friday you wanted to review the overall order of priority we give to dates, but that hasn't been done yet, so the prioritisation for example in "find" is still what it used to be, and is roughly: prefer print dates over electronic dates, prefer crossref then epmc then unpaywall, then fall back to the created or deposited dates in crossref record metadata if nothing else is available.

**Comment by JosephMcArthur** [comment]
Unless having all systems listen to unpaywall dates first is easy, please unassign this so we can focus elsewhere.
